### PR TITLE
v6.0.0-alpha.8

### PR DESCRIPTION
**Pull request by LukasTy**
- [x] Compare the last tag with the branch upon which you want to release (`next` for the alpha / beta releases and `master` for the current stable version).
      To do so, use `yarn release:changelog` The options are the following:

  ```bash
  yarn release:changelog
     --githubToken   YOUR_GITHUB_TOKEN (needs "public_repo" permission)
     --lastRelease   The release to compare against (default: the last one)
     --release       The branch to release (default: master)
  ```

  You can also provide the github token by setting `process.env.GITHUB_TOKEN` variable.

  In case of a problem, another method to generate the changelog is available at the end of this page.

- [x] Clean the generated changelog, to match the format of [https://github.com/mui/mui-x/releases](https://github.com/mui/mui-x/releases).
- [x] Update the root `package.json`'s version
- [x] Update the versions of the other `package.json` files and of the dependencies with `yarn release:version`.
- [x] Fix manually the package version in `x-date-pickers/package.json` and `x-date-pickers-pro/package.json`.
- [x] Open PR with changes and wait for review and green CI.
- [ ] Merge PR once CI is green, and it has been approved.

### Release the packages

- [ ] Checkout the last version of the working branch
- [ ] Make sure you have the latest dependencies installed: `yarn`.
- [ ] Build the packages: `yarn release:build`.
- [ ] Release the versions on npm: `yarn release:publish` (you need your 2FA device).
- [ ] Push the newly created tag: `yarn release:tag`.

### Publish the documentation

The documentation must be updated on the `docs-vX` branch (`docs-v4` for `v4.X` releases, `docs-v5` for `v5.X` releases, ...)

- [ ] Push the working branch on the documentation release branch to deploy the documentation with the latest changes.

```sh
git push upstream next:docs-next -f
```

You can follow the deployment process [on the Netlify Dashboard](https://app.netlify.com/sites/material-ui-x/deploys?filter=docs-v5)
Once deployed, it will be accessible at https://material-ui-x.netlify.app/ for the `docs-v5` deployment.

### Publish GitHub release

- [ ] After documentation is deployed, publish a new release on [GitHub releases page](https://github.com/mui/mui-x/releases)

### CODE REVIEW

File: packages/x-date-pickers/package.json [hunk 1]
```diff
@@ -48,9 +48,9 @@
     "@date-io/core": "^2.16.0",
     "@date-io/date-fns": "^2.16.0",
     "@date-io/date-fns-jalali": "^2.16.0",
-    "@date-io/jalaali": "^2.16.1",
-    "@date-io/hijri": "^2.16.1",
     "@date-io/dayjs": "^2.16.0",
+    "@date-io/hijri": "^2.16.1",
```
Comment (LukasTy) [inline packages/x-date-pickers/package.json:52]: All this package location shuffling—lerna or yarn doing it's job. 😉

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,75 @@
...
+
+_Nov 17, 2022_
+
+We'd like to offer a big thanks to the 11 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Support aggregating data from multiple row fields (#6656) @cherniavskii
+- 📚 Documentation improvements
+- 🐞 Bugfixes
+
+#### Breaking changes
```
Comment (flaviendelangle) [inline CHANGELOG.md:16]: The breaking change should be inside the pickers h3, just above the pickers changes h4

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,75 @@
...
+- [DataGridPremium] Support aggregating data from multiple row fields (#6656) @cherniavskii
+- [DataGridPro] Fix detail panel not working with `getRowSpacing` prop (#6707) @cherniavskii
+- [DataGridPro] Opt-out for column jump back on re-order (#6733) @m4theushw
+
+### `@mui/x-date-pickers@v6.0.0-alpha.8` / `@mui/x-date-pickers-pro@v6.0.0-alpha.8`
+
+#### Changes
+
+- [pickers] Fix typing and prop drilling on `DateRangeCalendar` and multi input range fields (#6852) @flaviendelangle
+- [pickers] Pass the ampm prop from the new pickers to their field (#6868) @flaviendelangle
```
Comment (flaviendelangle) [inline CHANGELOG.md:49]:
```suggestion
- [pickers] Pass the `ampm` prop from the new pickers to their field (#6868) @flaviendelangle
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,75 @@
...
+
+```diff
+-MuiClockPicker: {
++MuiTimeClock: {
+```
+### `@mui/x-data-grid@v6.0.0-alpha.8` / `@mui/x-data-grid-pro@v6.0.0-alpha.8` / `@mui/x-data-grid-premium@v6.0.0-alpha.8`
+
+#### Changes
+
+- [DataGrid] Fix ErrorOverlay not receiving defined input props (#6819) @banoth-ravinder
```
Comment (flaviendelangle) [inline CHANGELOG.md:35]:
```suggestion
- [DataGrid] Fix `ErrorOverlay` not receiving defined input props (#6819) @banoth-ravinder
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,75 @@
...
+#### Changes
+
+- [pickers] Fix typing and prop drilling on `DateRangeCalendar` and multi input range fields (#6852) @flaviendelangle
+- [pickers] Pass the ampm prop from the new pickers to their field (#6868) @flaviendelangle
+- [pickers] Rename `CalendarPickerView`, `ClockPickerView` and `CalendarOrClockPickerView` (#6855) @flaviendelangle
+- [pickers] Rename `ClockPicker` into `TimeClock` (#6851) @flaviendelangle
+
+### Docs
+
+- [docs] Add dayjs to the dependencies (#6862) @m4theushw
```
Comment (flaviendelangle) [inline CHANGELOG.md:55]:
```suggestion
- [docs] Add `dayjs` to the dependencies (#6862) @m4theushw
```

File: CHANGELOG.md (hunk 2 above)
Comment (LukasTy) [reply CHANGELOG.md:32]: Updated everything. 😉

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,76 @@
...
+#### Changes
+
+- [DataGrid] Fix `ErrorOverlay` not receiving defined input props (#6819) @banoth-ravinder
+- [DataGrid] Fix conflict with `onResize` added to `React.HTMLAttributes` (#6797) @vizv
+- [DataGrid] Make more `apiRef` methods private (#6700) @cherniavskii
+- [DataGrid] Provide a clear error message when upgrading (#6685) @oliviertassinari
+- [DataGridPremium] Allow to customize the indent of group expansion toggle (#6837) @MBilalShafi
+- [DataGridPremium] Support aggregating data from multiple row fields (#6656) @cherniavskii
+- [DataGridPro] Fix detail panel not working with `getRowSpacing` prop (#6707) @cherniavskii
+- [DataGridPro] Opt-out for column jump back on re-order (#6733) @m4theushw
```
Comment (cherniavskii) [inline CHANGELOG.md:27]:
```suggestion
- [DataGridPro] Opt-out for column jump back on re-order (#6733) @gavbrennan
```

**Review by cherniavskii — APPROVED** [review]

**Review by joserodolfofreitas — APPROVED** [review]

File: CHANGELOG.md [hunk 7]
```diff
@@ -18,20 +18,20 @@ We'd like to offer a big thanks to the 11 contributors who made this release pos
...
 
 - [DataGrid] Fix `ErrorOverlay` not receiving defined input props (#6819) @banoth-ravinder
-- [DataGrid] Fix conflict with `onResize` added to `React.HTMLAttributes` (#6797) @vizv
+- [DataGrid] Fix conflict with the latest version of `@types/react` (#6797) @vizv
 - [DataGrid] Make more `apiRef` methods private (#6700) @cherniavskii
 - [DataGrid] Provide a clear error message when upgrading (#6685) @oliviertassinari
 - [DataGridPremium] Allow to customize the indent of group expansion toggle (#6837) @MBilalShafi
 - [DataGridPremium] Support aggregating data from multiple row fields (#6656) @cherniavskii
-- [DataGridPro] Fix detail panel not working with `getRowSpacing` prop (#6707) @cherniavskii
+- [DataGridPro] Fix detail panel not working with `getRowSpacing` prop (#6707) @gavbrennan
```
Comment (cherniavskii) [inline CHANGELOG.md]: This PR was actually mine, my comment was referring to #6733

Comment (LukasTy) [reply CHANGELOG.md]: Yeah, sorry, spotted that mistake and did update it with [a later commit](https://github.com/mui/mui-x/pull/6894/commits/200bc5cf2cf710cb1b2c80eb416230940d87b5a9). 🙈 🤦

File: CHANGELOG.md [hunk 8]
```diff
@@ -3,6 +3,76 @@
...
+-<ClockPicker {...props} />
++<TimeClock {...props} />
+```
+
+Component name in the theme has changed as well:
+
+```diff
+-MuiClockPicker: {
++MuiTimeClock: {
+```
```
[the remark this answers is not in the document]
Comment (LukasTy) [reply CHANGELOG.md]: Thank you for spotting it 👍

**Pull request merged**